lib/repo_tools.pm

```diff
@@ -133,6 +154,30 @@ sub rmt_mirror_repo {
     assert_script_run 'rmt-cli repo list';
 }
 
+=head2 rmt_import_data
+    rmt_import_data($datafile);
+RMT server import data about available repositories and the mirrored packages 
+from disconnected RMT server, then verify imported repos on new RMT server.
+=cut
+sub rmt_import_data {
+    my ($datafile) = @_;
+    my $datapath = "/mnt/external/";
+    # Decompress the RMT data file to test path
+    assert_script_run("mkdir -m 777 $datapath");
```

Comment:
Is this the way customers will follow? I mean setting permissions this way? It's minor, but would be better if we follow guidelines we ship to the users.

Reply:
Changed to 'mkdir $datapath' and '-m 777' is not needed for my test.